modules/dnn/test/test_model.cpp

```diff
@@ -810,8 +810,8 @@ TEST_P(Test_Model, TextDetectionByEAST)
 
 TEST_P(Test_Model, YOLOv7)
 {
-    std::string weightPath = findDataFile("yolov7-not-simplified.onnx", true);
-    std::string imgPath = findDataFile("test5.jpg", true);
+    std::string weightPath = _tf("onnx/models/yolov7_not_simplified.onnx");
```

Comment:
@fengyuentau Which export arguments from the `yolov7` repository did you use to produce a working onnx model? :)

Reply:
This model is exported using the script from https://github.com/opencv/opencv/issues/22215#issuecomment-1190053395, which is not simplified so that some changes need to be done for OpenCV DNN to support.

You can get a carefully exported YOLOv7 already working with OpenCV DNN from [this comment](https://github.com/opencv/opencv/issues/22215#issuecomment-1190554236).

Reply:
Thank you! :)